dev-client/ios/Terraso LandPKS.xcodeproj/xcshareddata/xcschemes/Terraso LandPKS.xcscheme

```diff
@@ -41,7 +41,7 @@
       </Testables>
    </TestAction>
    <LaunchAction
-      buildConfiguration = "Release"
+      buildConfiguration = "Debug"
```

Comment:
commnet: @paulschreiber it seems like we had to make this change to get the app to build properly for development, does this make sense to you?

Reply:
What happens if it's set to release?

Reply:
if is set to release when working on dev environment, you don't get access to logs and other dev functionality